shapes/neurosciencegraph/datashapes/atlas/volumetricdatalayer/schema.json

```diff
@@ -44,56 +46,30 @@
               "maxCount": 1
             },
             {
-              "path": "nsg:layer",
-              "name": "layer",
-              "description": "layer for PH data layer",
-              "node": "https://neuroshapes.org/commons/typedlabeledontologyterm/shapes/LayerOntologyTermShape",
+              "path": "nsg:isRegisteredIn",
+              "class": "nsg:BrainAtlasSpatialReferenceSystem",
               "maxCount": 1
             },
             {
-              "path": "schema:distribution",
-              "name": "Distribution",
-              "description": "Volumetric data layer location.",
-              "node": "this:VolumetricDataLayerDistributionShape"
+              "path": "nsg:imageModality",
+              "name": "Image modality",
+              "description": "The modality of data capture (MRI, fMRI, EM, etc.)",
+              "datatype": "xsd:string",
+              "maxCount": 1
             },
             {
               "path": "nsg:brainLocation",
               "name": "brain location",
               "description": "Brain Location (brain region, atlas space).",
-              "node": "https://neuroshapes.org/commons/brainlocation/shapes/BrainLocationShape",
+              "node": "this:BrainLocationShape",
```

Comment:
This should be https://neuroshapes.org/commons/brainlocation/shapes/BrainLocationShape

Reply:
Here `this:BrainLocationShape` only enforce a `"minCount": 1` to the original `https://neuroshapes.org/commons/brainlocation/shapes/BrainLocationShape`. I personally think it's good to enforce it, though in a near future, this dataset will be linked to an `AtlasRelease`.

Reply:
Alright I see.